src/switchroot/ostree-prepare-root.c

```diff
@@ -449,12 +449,19 @@ main (int argc, char *argv[])
           if (!cfs_digest_buf)
             errx (EXIT_FAILURE, "Failed to query digest: %s", error->message);
 
-          expected_digest = g_malloc (OSTREE_SHA256_STRING_LEN + 1);
-          ot_bin2hex (expected_digest, cfs_digest_buf, g_variant_get_size (cfs_digest_v));
+          if (composefs_config->require_verity)
```

Comment:
I'm confused by this change which is underneath `if is_signed` which always implies `require_verity` right? 

I think I'd just add a `g_assert (composefs_config->require_verity)` here instead or so.